### PR TITLE
fix tf inc metric format

### PR DESCRIPTION
**Pull request by y199387**
## Description
  Fix Nano TensorFlow INC metric bug.

related issue
https://github.com/intel-analytics/BigDL/issues/5305

### 3. How to test?
- [x] Unit test
- [x] Jenkins
http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation/653/

### CODE REVIEW

**Comment by yangw1234** [comment]
@zhentaocc could you help review this?

**Comment by zhentaocc** [comment]
What's the bug?

**Comment by y199387** [comment]
> What's the bug?

According to Keras API doc, y_true should be first and y_pred after.
The current code may make the accuracy calculation wrong.
https://keras.io/api/metrics/accuracy_metrics/

File: python/nano/src/bigdl/nano/deps/neural_compressor/tensorflow/metric.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ class TensorflowINCMetric(BaseINCMetric):
     def stack(self, preds, labels):
 
         # calculate accuracy
-        preds = tf.stack(preds)
         labels = tf.stack(labels)
-        return preds, labels
+        preds = tf.stack(preds)
+        return labels, preds
```
Comment (zhentaocc) [inline python/nano/src/bigdl/nano/deps/neural_compressor/tensorflow/metric.py:26]: I think you should override the `result` function instead.
```suggestion
        preds = tf.stack(preds)
        labels = tf.stack(labels)
        return preds, labels

    def result(self):
        # calculate accuracy
        preds, labels = self.stack(self.pred_list, self.label_list)
        accuracy = self.metric(labels, preds)
        return self.to_scalar(accuracy)
```

File: python/nano/src/bigdl/nano/deps/neural_compressor/tensorflow/metric.py [hunk 2]
```diff
@@ -21,9 +21,16 @@ class TensorflowINCMetric(BaseINCMetric):
     def stack(self, preds, labels):
 
         # calculate accuracy
+        labels = tf.stack(labels)
```
Comment (zhentaocc) [inline python/nano/src/bigdl/nano/deps/neural_compressor/tensorflow/metric.py:24]: This line is redundant.
